packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -49,49 +54,45 @@ class Neo4jGraphQL {
 
     private driver?: Driver;
 
-    private driverConfig?: DriverConfig;
+    public config?: Neo4jGraphQLConfig;
 
     // eslint-disable-next-line @typescript-eslint/no-unused-vars,class-methods-use-this
     debug(message: string): void {
         return undefined;
     }
 
     constructor(input: Neo4jGraphQLConstructor) {
-        this.driver = input.driver;
-        this.driverConfig = input.driverConfig;
+        const { config = {}, debug, driver, ...rest } = input;
```

Comment:
`rest` isn't a very clear variable name here, could `schemaDefinition` be a little clearer given that it's basically an instance of `IExecutableSchemaDefinition`?

Reply:
Sure